src/style.css

```diff
@@ -0,0 +1,140 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+h2 p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.page__product-card {
+  width: 200px;
+}
+
+.product-card {
+  padding-top: 34px;
+  padding-left: 16px;
+  padding-right: 16px;
+  flex-direction: column;
+  display: flex;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__img {
+  margin-bottom: 40px;
+  height: 134px;
+  width: 100%;
+  padding: 3px;
```

Comment:
You should add only left and right margins
Or set fixed width and use margin: 0 auto 40px;